lighthouse-cli/chrome-finder.ts

```diff
@@ -75,14 +79,59 @@ export function darwin() {
 }
 
 export function linux() {
+  let installations: Array<string> = [];
+
+  // Look into LIGHTHOUSE_CHROMIUM_PATH
   const execPath = process.env.LIGHTHOUSE_CHROMIUM_PATH;
```

Comment:
since we are trying 3 different techniques to find the binary we probably shouldn't call this first one `execPath`. IMO we dont really need to use the variable.